src/roktManager.ts

```diff
@@ -111,7 +121,23 @@ export default class RoktManager {
         this.processMessageQueue();
     }
 
-    public selectPlacements(options: IRoktSelectPlacementsOptions): Promise<IRoktSelection> {
+    /**
+     * Selects placements based on the provided options.
+     * This method handles user identity updates and attribute mapping before selecting placements.
+     * 
+     * @param {IRoktSelectPlacementsOptions} options - The options for selecting placements, including attributes and optional identifier
+     * @returns {Promise<IRoktSelection>} A promise that resolves to the selected placements
+     * 
+     * @example
+     * // Correct usage with await
+     * await roktManager.selectPlacements({
```

Comment:
I think this might be the more accurate usage.
```suggestion
     * await window.mParticle.Rokt.selectPlacements({
```